foundation_cms/static/scss/redesign_main.scss

```
@@ -3,6 +3,7 @@
 @import "../scss/fonts";
 @import "../scss/components/primary_nav";
 @import "../scss/components/footer";
+@use "../scss/components/images.scss";
```

```suggestion
@import "../scss/components/images.scss";
```
@mmmavis just had a PR where this is changed to `@import`, so figure we should catch that now!